packages/create-svelte/templates/default/package.template.json

```diff
@@ -5,7 +5,8 @@
 		"dev": "svelte-kit dev",
 		"build": "svelte-kit build",
 		"package": "svelte-kit package",
-		"preview": "svelte-kit preview"
+		"preview": "svelte-kit preview",
+		"prepare": "svelte-kit sync"
```

Comment:
I would think that we should call them both `prepare` or call them both `sync`

Reply:
Calling the script `prepare` is required so that it gets automatically run at the correct time by npm. I don't know that we also want that to inform what we call the CLI command.

Reply:
Ah, I thought `build` invoke `sync` via code: https://github.com/sveltejs/kit/pull/4182/files#diff-09b70580810ae66bf783d467b3ea4f54cd1ac4cb57532b38b788145e27cc0983R26

Does this mean it will be run twice?

Reply:
`prepare` only runs (automatically) at certain points, like `install`: https://docs.npmjs.com/cli/v8/using-npm/scripts#life-cycle-scripts.